# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/notification/Notification.h
# Commit: bde8ca8fd963
# Full Hash: bde8ca8fd9636bc6650ee6768dfb8e60fc1da596
# Author: Kagami Sascha Rosylight <saschanaz@outlook.com>
# Date: 2024-11-12 09:32:50
# Regressor Bug: 1928702
# File Overlap Count: 1
# Description:
#   Bug 1928702 - Part 6: Move NotificationObserver functions to NotificationUtils r=asuth
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D227691
# ==============================================================================

diff -r fca3921d09a5 -r bde8ca8fd963 dom/notification/Notification.h
--- a/dom/notification/Notification.h	Mon Nov 11 20:34:13 2024 +0000
+++ b/dom/notification/Notification.h	Mon Nov 11 20:34:13 2024 +0000
@@ -226,9 +226,6 @@
 
   bool DispatchClickEvent();
 
-  static nsresult RemovePermission(nsIPrincipal* aPrincipal);
-  static nsresult OpenSettings(nsIPrincipal* aPrincipal);
-
   nsresult DispatchToMainThread(already_AddRefed<nsIRunnable>&& aRunnable);
 
  protected: